tests/hello.ts

```diff
@@ -1,9 +0,0 @@
-import { logInfo } from "@suborbital/suborbital"
```

Comment:
Is there a reason for removing the test?

Reply:
It's not a Go test and it wasn't referenced everywhere as far as I can tell.